### PR TITLE
Feature/fp21 46 grids area sorting

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**
# Description
Created logic for sorting tools  component by alphabet and date on frontend,  and added style with responsiveness.

# How to test?
To test by recently added - change in a table "created_at" date for some tools to see the result.
Can be tested in storybook and in the browser.
# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/Sorting/Sorting.component.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              src="/assets/vectors/vector-arrow-down.svg"
+              alt="Arrow down to open sorting dropdown menu"
+            />
+          )}
+        </button>
+        {isOpen ? (
+          <div className="sorting-options-area sorting-area-general">
+            <button
+              onClick={(e) => {
+                props.setSelected(e.target.value);
```
Comment (shpomp) [inline packages/client/src/components/Sorting/Sorting.component.js:33]:
```suggestion
                props.setSelected(e.target.value);
                 setIsOpen(!isOpen);
```
I would consider adding this here, just to have the dropdown hidden once the sorting is selected, but that is a matter of opinion, not an actual problem.

File: packages/client/src/components/Sorting/Sorting.component.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+              }}
+              value="a-z"
+              type="button"
+              className="sorting-button-general sorting-options-button"
+            >
+              A-Z
+            </button>
+            <button
+              onClick={(e) => {
+                props.setSelected(e.target.value);
```
Comment (shpomp) [inline packages/client/src/components/Sorting/Sorting.component.js:44]:
```suggestion
                props.setSelected(e.target.value);
                 setIsOpen(!isOpen);
```
I would consider adding this here, just to have the dropdown hidden once the sorting is selected, but that is a matter of opinion, not an actual problem.

File: packages/client/src/components/ToolItem/ToolItem.component.js [hunk 3]
```diff
@@ -86,10 +87,7 @@ export const ToolItem = ({
           <h6 className="product-title">{title}</h6>
         </div>
         <div className="title-image">
-          <img
-            src="/assets/images/tool-pic-stinky-fish.png"
-            alt="stinky-fish"
-          />
+          <img src={picture} alt="stinky-fish" />
```
Comment (shpomp) [inline packages/client/src/components/ToolItem/ToolItem.component.js]: The `alt="stinky-fish"` will not fit every tool, so I would suggest working with the picture URL to extract the name somewhat dynamically.
Something like:
Take `picture` string, cut away the part of `/assets/images/tool-pic-` and the `.png` at the end, and do something like "Picture of tool" + `result`, where result is what you have after processing the picture string. Hope that makes sense 😅

**Review by shpomp — APPROVED** [review]
Very nice 🤩

File: packages/client/src/components/ToolItem/ToolItem.component.js (hunk 3 above)
Comment (SvitlanaBurlaka) [reply packages/client/src/components/ToolItem/ToolItem.component.js]: Yes, I agree with you. It makes sense :) Fixed already .

File: packages/client/src/components/Sorting/Sorting.component.js (hunk 2 above)
Comment (SvitlanaBurlaka) [reply packages/client/src/components/Sorting/Sorting.component.js:44]: Fixed that too :)

**Review by rebeccahockley — APPROVED** [review]
Looks great guys! Really solid code :D

**Pull request merged**